Documentation/ContentElements/CreatingContent/Index.rst

```diff
@@ -27,6 +27,12 @@ itself already provides mostly self-explanatory descriptions.
 Types of content elements
 =========================
 
+If you are working with the *Introduction Package*, you will see
+more content elements than described here, because the *Introduction
```

Comment:
Should rather be "Bootstrap package comes with several content elements ...", not Introduction Package.